run.py

```diff
@@ -1,8 +1,8 @@
 import os
 
+import settings
 from app.app import app
 
-
```

Comment:
dalczego nie dwie linie po importach?

Reply:
black/flake8 automatycznie mi zmienia na jedną linię (?). A z blackiem to ja się nie kłócę :C